packages/marko-web-theme-monorail/components/blocks/section-forbes.marko

```diff
@@ -27,7 +30,15 @@ $ const blockName = "section-forbes";
 
 $ const rowStyle = (input.reverse) ? "flex-direction: row-reverse" : "";
 
-<marko-web-query|{ nodes, section: querySection }| name="website-scheduled-content" params=queryParams>
+<marko-web-query|{ nodes: returnedNodes, section: querySection }| name="website-scheduled-content" params=queryParams>
+  $ const nodes = withSponsored ? returnedNodes.map((node) => {
```

Comment:
@Shinsina - since you're using this same function in two places, and may need it elsewhere, I would suggest creating as a utility (utils folder) and importing into this component.  Likely would need to pass in the label you're searching for ("Sponsored"), as well as the section name you want to set (also "Sponsored" here) based on the presence of that label.

Reply:
@brandonbk Utility function has been added as `utils/set-section-name-from-label` and has been used in place of this duplicated behavior where applicable.